src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -335,6 +348,16 @@ public function getAccessDecisionLog()
         return $this->data['access_decision_log'];
     }
 
+    /**
+     * Returns the log of the votes processed in the access decision manager.
+     *
+     * @return Data|array
+     */
+    public function getVoterDetails(): iterable
```

Comment:
This is always an array, isn't it?

Reply:
I don't think so, original data seems to be transformed into a Data object when lateCollect is invoked. Are you agree with that ?

Reply:
I don't think that's needed. Thanks for checking it btw. :+1: